test/e2e/apps/cronjob.go

```diff
@@ -481,10 +482,7 @@ var _ = SIGDescribe("CronJob", func() {
 		cjs, err = cjClient.List(ctx, metav1.ListOptions{LabelSelector: "special-label=" + f.UniqueName})
 		framework.ExpectNoError(err)
 		// Should have <= 2 items since some cronjobs might not have been deleted yet due to finalizers
-		if len(cjs.Items) > 2 {
-			framework.Logf("got unexpected filtered list: %v", cjs.Items)
-			framework.Fail("filtered list should be <= 2")
-		}
+		gomega.Expect(len(cjs.Items)).To(gomega.BeNumerically("<=", 2), "filtered list should be <= 2")
```

Comment:
The list content is no longer available.

```suggestion
		gomega.Expect(len(cjs.Items)).To(gomega.BeNumerically("<=", 2), "filtered list length should be <= 2, got:\n%s", format.Object(cjs.Items, 1))
```

where "format" is gomega/format.